BranchMetrics/mparticle-apple-integration-branchmetrics
# [INTENG-18507] Update to latest Branch SDK v2.1.2

## Reference
INTENG-18507 -- Update mParticle kit to new BranchSDK version

## Summary
Updated the Branch SDK to the latest version (2.1.2) and changed all imports from `Branch` to `BranchSDK`

## Motivation
To use the latest version of mParticle-Kit, the SDK needs to be updated to the latest Branch version.

## Type Of Change


## Testing Instructions

Setup and run the mParticle test app and ensure all of the versions are correct and the functionality of the kit remains the same.









cc @BranchMetrics/saas-sdk-devs for visibility.
